features/step_definitions/filtering_steps.rb

```diff
@@ -553,8 +553,8 @@
 
 And(/^I select some document types$/) do
   click_on('Document type')
-  find('.govuk-label', text: 'Policy papers').click
-  find('.govuk-label', text: 'Consultations (closed)').click
+  find('.govuk-checkboxes__item .govuk-label', text: 'Policy papers').click
+  find('.govuk-checkboxes__item .govuk-label', text: 'Consultations (closed)').click
```

Comment:
How is this failing test related to the changes you made? Is this just a flickering test?

Reply:
Capybara is being told to look for a `.govuk-label` that contains the text 'Policy papers'. We've just added that text into the label for the search field, so since Capybara is being vague about its matches, it's finding both of them. This change makes the CSS selector more specific so it can only match a checkbox label, which is what it's supposed to do.

Reply:
😡Capybara trying to be too helpful!!! 

Makes sense.